### PR TITLE
[TI MISP] Add datastream for Attributes API endpoint

### LINKED ISSUE #4115: [MISP] Support Attributes API
Our [MISP integration](https://github.com/elastic/integrations/tree/main/packages/ti_misp) currently supports the events API to ingest IOC's. However, the original MISP filebeat module supported both the events and attributes API. The events API generally exposes more data, but can lead to increased storage costs for data which users may not necessarily need or want. The attributes API allows for better filtering to ensure users only ingest the IOC's they are interested in. 

Can we add support for [MISP Attributes ](https://www.misp-project.org/openapi/#tag/Attributes) as an additional data stream within the MISP integration please?

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Add new datastream to support the `/attributes/restSearch` API endpoint

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Closes #4115

### CODE REVIEW

**Comment by P1llus** [comment]
This is a great contribution @legoguy1000 . Please ping me when you feel its finished.

**Comment by legoguy1000** [comment]
Will do. I think it's pretty much done, the only thing I've been debating is whether to align the fields to match the other datastream or not? There is overlap in the data that's retrieved but it's structure is different and I've been debating whether to rename the fields to match the other datastream or just leave as is.

**Comment by legoguy1000** [comment]
:)

**Comment by ebeahan** [comment]
thanks for the bump, @legoguy1000!

If you're ready to have this one looked at again, can you move it out of draft?

File: packages/ti_misp/_dev/build/docs/README.md [hunk 1]
```diff
@@ -13,4 +13,12 @@ The filters themselves are based on the [MISP API documentation](https://www.cir
...
 {{fields "threat"}}
 
-{{event "threat"}}
\ No newline at end of file
+{{event "threat"}}
+
+### Threat Attributes
+
+The MISP integration configuration allows to set the polling interval, how far back it should look initially, and optionally any filters used to filter the results.
+This data stream uses the `/attributes/restSearch` API endpoint which returns a more granual information regarding MISP attributes and additional information
```
Comment (andrewkroh) [inline packages/ti_misp/_dev/build/docs/README.md:21]:
```suggestion
This data stream uses the `/attributes/restSearch` API endpoint which returns more granular information regarding MISP attributes and additional information.
```

**Comment by P1llus** [comment]
We could safely take over this PR. The current progress is that we need to review the integration to make sure it produces the same outcome as the existing datastream (it just uses two different API's, this one is less performance heavy).

We should also update the documentation to ensure that they only use one or the other.

File: packages/ti_misp/_dev/build/docs/README.md (hunk 1 above)
Comment (kcreddy) [reply packages/ti_misp/_dev/build/docs/README.md]: Updated the doc

File: packages/ti_misp/data_stream/threat/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -37,68 +37,67 @@ processors:
...
+        - misp.event.Object.Attribute.Galaxy
+        - misp.event.Object.Attribute.ShadowAttribute
       ignore_missing: true
   - remove:
       field:
-        - misp.Attribute
+        - misp.event.Attribute
       ignore_missing: true
-      if: ctx.misp?.Attribute != null && ctx.misp.Attribute.size() == 0
+      if: 'ctx.misp?.event?.Attribute != null && ctx.misp?.event?.Attribute.size() == 0'
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat/elasticsearch/ingest_pipeline/default.yml:68]:
```suggestion
      if: 'ctx.misp?.event?.Attribute != null && ctx.misp.event.Attribute.size() == 0'
```

File: packages/ti_misp/data_stream/threat/elasticsearch/ingest_pipeline/default.yml [hunk 3]
```diff
@@ -37,68 +37,67 @@ processors:
...
   - remove:
       field:
-        - misp.Object
+        - misp.event.Object
       ignore_missing: true
-      if: ctx.misp?.Object != null && ctx.misp.Object.size() == 0
+      if: 'ctx.misp?.event?.Object != null && ctx.misp?.event?.Object.size() == 0'
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat/elasticsearch/ingest_pipeline/default.yml:73]:
```suggestion
      if: 'ctx.misp?.event?.Object != null && ctx.misp.event.Object.size() == 0'
```

File: packages/ti_misp/data_stream/threat/elasticsearch/ingest_pipeline/default.yml [hunk 4]
```diff
@@ -37,68 +37,67 @@ processors:
       ignore_missing: true
   - rename:
       field: json.Event
-      target_field: misp
+      target_field: misp.event
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat/elasticsearch/ingest_pipeline/default.yml:40]: What was the reason for moving to `misp.event`?

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 5]
```diff
@@ -0,0 +1,396 @@
...
+      field: misp.event.analysis
+      type: long
+      target_field: threat.indicator.scanner_stats
+      ignore_missing: true
+
+  ## File/Hash indicator operations
+  - set:
+      field: threat.indicator.type
+      value: file
+      if: "ctx.misp?.attribute?.type != null && (['md5', 'impfuzzy', 'imphash', 'pehash', 'sha1', 'sha224', 'sha256', 'sha3-224', 'sha3-256', 'sha3-384', 'sha3-512', 'sha384', 'sha512', 'sha512/224', 'sha512/256', 'ssdeep', 'tlsh', 'vhash'].contains(ctx.misp?.attribute?.type) || ctx.misp?.attribute?.type.startsWith('filename'))"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:141]:
```suggestion
      if: "ctx.misp?.attribute?.type != null && (['md5', 'impfuzzy', 'imphash', 'pehash', 'sha1', 'sha224', 'sha256', 'sha3-224', 'sha3-256', 'sha3-384', 'sha3-512', 'sha384', 'sha512', 'sha512/224', 'sha512/256', 'ssdeep', 'tlsh', 'vhash'].contains(ctx.misp.attribute.type) || ctx.misp.attribute.type.startsWith('filename'))"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 6]
```diff
@@ -0,0 +1,396 @@
...
+  - rename:
+      field: misp.attribute.value
+      target_field: "threat.indicator.file.hash.{{misp.attribute.type}}"
+      ignore_missing: true
+      if: "ctx.threat?.indicator?.type == 'file' && ctx.misp?.attribute?.type != null && !ctx.misp?.attribute?.type.startsWith('filename')"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:146]:
```suggestion
      if: "ctx.threat?.indicator?.type == 'file' && ctx.misp?.attribute?.type != null && !ctx.misp.attribute.type.startsWith('filename')"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 7]
```diff
@@ -0,0 +1,396 @@
...
+      field: misp.attribute.value
+      target_field: threat.indicator.file.name
+      ignore_missing: true
+      if: "ctx.threat?.indicator?.type == 'file' && ctx.misp?.attribute?.type == 'filename'"
+  - grok:
+      field: misp.attribute.type
+      patterns:
+        - "%{WORD}\\|%{WORD:_tmp.hashtype}"
+      ignore_missing: true
+      if: ctx.misp?.attribute?.type != null && ctx.misp?.attribute?.type.startsWith('filename|')
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:157]:
```suggestion
      if: ctx.misp?.attribute?.type != null && ctx.misp.attribute.type.startsWith('filename|')
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 8]
```diff
@@ -0,0 +1,396 @@
...
+  - grok:
+      field: misp.attribute.value
+      patterns:
+        - "%{DATA:threat.indicator.file.name}\\|%{GREEDYDATA:_tmp.hashvalue}"
+      ignore_missing: true
+      if: ctx.misp?.attribute?.type != null && ctx.misp?.attribute?.type.startsWith('filename|')
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:163]:
```suggestion
      if: ctx.misp?.attribute?.type != null && ctx.misp?.attribute?.type.startsWith('filename|')
```
```suggestion
      if: ctx.misp?.attribute?.type != null && ctx.misp.attribute.type.startsWith('filename|')
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 9]
```diff
@@ -0,0 +1,396 @@
...
+  - set:
+      field: threat.indicator.file.hash.{{_tmp.hashtype}}
+      value: "{{_tmp.hashvalue}}"
+      if: "ctx.misp?.attribute?.type != null && ctx.misp?.attribute?.type.startsWith('filename|') && ctx?._tmp?.hashvalue != null && ctx?._tmp?.hashtype != null"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:167]:
```suggestion
      if: "ctx.misp?.attribute?.type != null && ctx.misp.attribute.type.startsWith('filename|') && ctx._tmp?.hashvalue != null && ctx._tmp.hashtype != null"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 10]
```diff
@@ -0,0 +1,396 @@
...
+
+  ## URL/URI indicator operations
+  - set:
+      field: threat.indicator.type
+      value: url
+      if: "ctx.misp?.attribute?.type != null && ['url', 'link', 'uri'].contains(ctx.misp?.attribute?.type)"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:173]:
```suggestion
      if: "ctx.misp?.attribute?.type != null && ['url', 'link', 'uri'].contains(ctx.misp.attribute.type)"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 11]
```diff
@@ -0,0 +1,396 @@
...
+      field: threat.indicator.url.full
+      value: "{{{threat.indicator.url.original}}}"
+      ignore_empty_value: true
+      if: "ctx.threat?.indicator?.type == 'url' && ctx.misp?.attribute?.type != 'uri'"
+
+  ## Regkey indicator operations
+  - set:
+      field: threat.indicator.type
+      value: windows-registry-key
+      if: "ctx.misp?.attribute?.type != null && ctx.misp?.attribute?.type.startsWith('regkey')"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:190]:
```suggestion
      if: "ctx.misp?.attribute?.type != null && ctx.misp.attribute.type.startsWith('regkey')"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 12]
```diff
@@ -0,0 +1,396 @@
...
+      type: long
+      target_field: threat.indicator.as.number
+      ignore_missing: true
+      if: ctx.threat?.indicator?.type == 'autonomous-system'
+
+  ## Domain/IP/Port indicator operations
+  - set:
+      field: threat.indicator.type
+      value: domain-name
+      if: "ctx.misp?.attribute?.type != null && (ctx.misp?.attribute?.type == 'hostname' || ctx.misp?.attribute?.type.startsWith('domain'))"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:219]:
```suggestion
      if: "ctx.misp?.attribute?.type != null && (ctx.misp.attribute.type == 'hostname' || ctx.misp.attribute.type.startsWith('domain'))"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 13]
```diff
@@ -0,0 +1,396 @@
...
+  - set:
+      field: threat.indicator.type
+      value: ipv4-addr
+      if: "ctx.misp?.attribute?.type != null && ['ip-src', 'ip-src|port', 'ip-dst', 'ip-dst|port'].contains(ctx.misp?.attribute?.type)"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:223]:
```suggestion
      if: "ctx.misp?.attribute?.type != null && ['ip-src', 'ip-src|port', 'ip-dst', 'ip-dst|port'].contains(ctx.misp.attribute.type)"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 14]
```diff
@@ -0,0 +1,396 @@
...
+  - rename:
+      field: misp.attribute.value
+      target_field: threat.indicator.url.domain
+      ignore_missing: true
+      if: "ctx.threat?.indicator?.type == 'domain-name' && ctx.misp?.attribute?.type != 'domain|ip' && ctx.threat?.indicator?.url?.domain == null"
+  - rename:
+      field: misp.attribute.value
+      target_field: threat.indicator.ip
+      ignore_missing: true
+      if: "ctx.threat?.indicator?.type == 'ipv4-addr' && ctx.misp?.attribute?.type != 'domain|ip' && !['ip-src|port', 'ip-dst|port'].contains(ctx.misp?.attribute?.type)"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:233]: `ctx.misp?.attribute?.type != 'domain|ip' && !['ip-src|port', 'ip-dst|port'].contains(ctx.misp?.attribute?.type)` evaluates to true if `ctx.misp?.attribute?.type` is null. Is this intended?

Also note that that expression can be rewritten as `!['domain|ip', 'ip-src|port', 'ip-dst|port'].contains(ctx.misp?.attribute?.type)` by De Morgan.

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 15]
```diff
@@ -0,0 +1,396 @@
...
+        - "%{IP:threat.indicator.ip}\\|%{NUMBER:threat.indicator.port}"
+      ignore_missing: true
+      if: "['ip-src|port', 'ip-dst|port'].contains(ctx.misp?.attribute?.type)"
+
+  ## Email indicator operations
+  # Currently this ignores email-message, except setting the type it will leave the rest of the fields under misp.
+  - set:
+      field: threat.indicator.type
+      value: email-addr
+      if: "ctx.misp?.attribute?.type != null && ['email-dst', 'email-src'].contains(ctx.misp?.attribute?.type)"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:252]:
```suggestion
      if: "ctx.misp?.attribute?.type != null && ['email-dst', 'email-src'].contains(ctx.misp.attribute.type)"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 16]
```diff
@@ -0,0 +1,396 @@
...
+  - set:
+      field: threat.indicator.type
+      value: email-message
+      if: "ctx.misp?.attribute?.type != null && ctx.misp?.attribute?.type.startsWith('email') && !['email-dst', 'email-src'].contains(ctx.misp?.attribute?.type)"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:256]:
```suggestion
      if: "ctx.misp?.attribute?.type != null && ctx.misp.attribute.type.startsWith('email') && !['email-dst', 'email-src'].contains(ctx.misp.attribute.type)"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 17]
```diff
@@ -0,0 +1,396 @@
...
+  - append:
+      field: user.roles
+      value: "reporting_user"
+      if: ctx?.user?.email != null
+
+  ## MAC Address indicator operations
+  - set:
+      field: threat.indicator.type
+      value: mac-addr
+      if: "ctx.misp?.attribute?.type != null && ['mac-address', 'mac-eui-64'].contains(ctx.misp?.attribute?.type)"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:275]:
```suggestion
      if: "ctx.misp?.attribute?.type != null && ['mac-address', 'mac-eui-64'].contains(ctx.misp.attribute.type)"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 18]
```diff
@@ -0,0 +1,396 @@
...
+      field: misp.attribute.value
+      target_field: threat.indicator.mac
+      ignore_missing: true
+      if: ctx.threat?.indicator?.type == 'mac-addr'
+
+  ## Artifact indicator operations
+  - set:
+      field: threat.indicator.type
+      value: artifact
+      if: "ctx.misp?.attribute?.type != null && ctx.misp?.attribute?.type == 'mime-type'"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:286]:
```suggestion
      if: "ctx.misp?.attribute?.type == 'mime-type'"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 19]
```diff
@@ -0,0 +1,396 @@
...
+
+  ## Mutex indicator operations
+  - set:
+      field: threat.indicator.type
+      value: mutex
+      if: "ctx.misp?.attribute?.type != null && ctx.misp?.attribute?.type == 'mutex'"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:292]:
```suggestion
      if: "ctx.misp?.attribute?.type == 'mutex'"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml [hunk 20]
```diff
@@ -0,0 +1,396 @@
...
+
+  ## Software indicator operations
+  - set:
+      field: threat.indicator.type
+      value: software
+      if: "ctx.misp?.attribute?.type != null && ctx.misp?.attribute?.type == 'cpe'"
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:298]:
```suggestion
      if: "ctx.misp?.attribute?.type == 'cpe'"
```

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 14 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:233]: > ctx.misp?.attribute?.type != 'domain|ip' && !['ip-src|port', 'ip-dst|port'].contains(ctx.misp?.attribute?.type) evaluates to true if ctx.misp?.attribute?.type is null. Is this intended?

So, the expression before that `ctx.threat?.indicator?.type == 'ipv4-addr' &&` should mean that `ctx.misp?.attribute?.type` is not null

Comment (efd6) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml:233]: They are different fields; the first part of the total expression is a comparison with `ctx.threat?.indicator?.type` but the second parts are against `ctx.misp?.attribute?.type`. The logic of the pipeline may indicate that one implies the other, but it's not intrinsically guaranteed.

File: packages/ti_misp/data_stream/threat/elasticsearch/ingest_pipeline/default.yml (hunk 3 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 5 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 6 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 7 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 8 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 9 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 10 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 11 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 13 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 14 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Makes sense! Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 15 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 16 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 17 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 18 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 19 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

File: packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml (hunk 12 above)
Comment (kcreddy) [reply packages/ti_misp/data_stream/threat_attributes/elasticsearch/ingest_pipeline/default.yml]: Updated as suggested

**Review by efd6 — APPROVED** [review]

**Pull request merged**